docs/develprocess.md

```diff
@@ -858,6 +858,91 @@ genisoimage -o $OUTPUT -v -r -J --joliet-long -V "XCP-ng $VERSION" -c boot/isoli
 isohybrid --uefi $OUTPUT
 ```
 
+## Create unattended ISO file
```

Comment:
I think this whole section should belong to install.md (and be merged with the existing docs that cover PXE install so that we have one doc for automated install covering both PXE and modified ISO).

And also link to it from somwhere in `## ISO modification` to help users find it if they started reading the custom ISO docs with the goal of making an automated install, maybe between `### Contents of the installation ISO image` and `Modify the installer itself`.